File: starter-bots/JavaBot/src/main/java/Models/PlayerAction.java
package Models;

import Enums.*;
import java.util.*;

public class PlayerAction {

  public UUID playerId;
  public PlayerActions action;
  public int heading;

  public UUID getPlayerId() {
    return playerId;
  }

  public void setPlayerId(UUID playerId) {
    this.playerId = playerId;
  }

  public PlayerActions getAction() {
    return action;
  }

  public void setAction(PlayerActions action) {
    this.action = action;
  }

  public int getHeading() {
    return heading;
  }

  public void setHeading(int heading) {
    this.heading = heading;
  }
}

File: starter-bots/NETCoreBot/NETCoreBot/Models/PlayerAction.cs
using NETCoreBot.Enums;
using System;

namespace NETCoreBot.Models
{
    public class PlayerAction
    {
        public Guid PlayerId { get; set; }
        public PlayerActions Action { get; set; }
        public int? Heading { get; set; }
    }
}
